scripts/build-java-client.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# This script is intended to be run under the docker container at $ARMADADIR/build/java-client
+
+ROOT=$(pwd)
+JDIR=client/java
+cd $ROOT/$JDIR
+
+# Build the Java client using Maven
+mvn clean install
+
+if [ $? -eq 0 ]; then
+  jarfile=$(find target -type f -name 'java-client*.jar')
```

Comment:
This finds multiple jars if there are sources and javadoc jars as well (the case for the Scala client):
https://github.com/armadaproject/armada/blob/daac225a8549d98bb588213d9314c47df54a6fe5/scripts/build-scala-client.sh#L16-L17

Reply:
Yes, they are all in the `target` dir. Thanks!